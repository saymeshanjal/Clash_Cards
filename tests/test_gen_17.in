21
2 57 4
8 139 2
2 84 4
8 100 5
12 65 4
6 211 4
6 92 4
16 231 1
10 6 4
8 70 1
4 19 2
8 185 3
2 17 2
12 121 3
8 7 5
14 145 2
2 36 4
10 58 2
4 35 1
10 250 1
10 73 4