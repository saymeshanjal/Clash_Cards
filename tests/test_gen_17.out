0 2 0
20 21 29
0 2 0
8 62 0
208 441 275
1 14 5
1 14 5
6435 6435 0
0 235 17
35 35 0
1 2 3
27 43 0
0 1 1
430 494 0
0 70 0
1078 1079 1275
0 2 0
76 77 99
3 3 0
126 126 0
44 133 75
